CMakeLists.txt

```diff
@@ -73,22 +73,24 @@ macro(Cabana_add_dependency)
   set(Cabana_ENABLE_${CABANA_DEPENDENCY_OPTION} ${${CABANA_DEPENDENCY_PACKAGE}_FOUND})
 endmacro()
 
+# FIXME: HYPRE workaround. Hypre is currently not exporting the find_package(
+# MPI ) to find the MPI::MPI_C targets and therefore we add this here for now
+# to ensure those targets are found for HYPRE.
+if ( Cabana_REQUIRE_HYPRE )
```

Comment:
Should be ENABLE?

Reply:
Add a link to  hypre-space/hypre#423 in the comment.

Reply:
I just enabled it directly for now